### PR TITLE
Text cleanup in sidebar and main page

### PR DESCRIPTION
**Pull request by kmid5280**
Contributes to issue #428 . Made capitalizations in sidebar more consistent. Made descriptive text in intro template better match parameter names. Other assorted cleanup.

### CODE REVIEW

File: src/chime_dash/app/templates/en/sidebar.yml [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 
 severity_parameters: Severity Parameters
-hospitalized_rate: Hospitalization %(total infections)
-icu_rate: ICU %(total infections)
-ventilated_rate: Ventilated %(total infections)
-infectious_days: Infectious Days
-hospitalized_los: Average Hospital Length of Stay (days)
-icu_los: Average Days in ICU
-ventilated_los: Average Days on Ventilator
+hospitalized_rate: Hospitalization rate (% of total infections)
```
Comment (jlubken) [inline src/chime_dash/app/templates/en/sidebar.yml:12]: Rates are parameters between 0.0 and 1.0. Percents are only UI presentations of rates between 0.0 and 100.0.

**Review by jlubken — CHANGES_REQUESTED** [review]
If, you don't mind, please update the text in the penn_chime/cli.py and penn_chime/presentation.py sidebar to match.

File: src/chime_dash/app/templates/en/sidebar.yml (hunk 1 above)
Comment (kmid5280) [reply src/chime_dash/app/templates/en/sidebar.yml:12]: @jlubken Are you wanting me to change it back to say "Hospitalization rate (%)"?

**Comment by kmid5280** [comment]
@jlubken I pushed some additional changes. Is this what you were looking for?

**Review by jlubken — APPROVED** [review]

**Comment by kmid5280** [comment]
@jlubken Did you want me to change that line in src/chime_dash/app/templates/en/sidebar.yml as well? The one that says `hospitalized_rate: Hospitalization rate (% of total infections)` . Want to make sure the percent sign is correct.

**Pull request merged**